snap/snapcraft.yaml

```diff
@@ -143,6 +166,9 @@ parts:
     - -usr/share/doc/libfreetype6/TODO
     - -usr/share/doc/libfreetype6/changelog.Debian.gz
     - -usr/share/doc/libfreetype6/pcf/README
+    - -lib/$SNAPCRAFT_ARCH_TRIPLET/libexpat.so.1.6.7
```

Comment:
Same here, why these 3 are being unstaged now?

Reply:
Same as above. Just to make it build really

Reply:
Alright